CMakeLists.txt

```diff
@@ -251,7 +251,10 @@ include(cmake/SanitizerHelper.cmake)
 # Find required packages
 # =============================================================================
 find_package(BISON REQUIRED)
+find_package(Boost REQUIRED)
 find_package(FLEX 2.6 REQUIRED)
+cpp_cc_git_submodule(range-v3 BUILD PACKAGE range-v3 REQUIRED)
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/external/range-v3)
```

Comment:
Need to tweak this a bit. It should work with an external range-v3 too. In fact that's the only way I've had it working so far, because range-v3 CMake uses generator expressions and our CMake-to-Makefile hackery isn't sophisticated enough.